_migration-assistant/migration-console/migration-console-commands-references.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 title: Command reference
-nav_order: 35
+nav_order: 40
```

Comment:
Add a redirect from migration-console

Reply:
Please add a redirect for all /index/ subpages. For example, in the /migration-phases/index.md file, add a redirect from /migration-phases/, etc.